lib/utils/seqid/src/test/test.js

```diff
@@ -20,57 +17,96 @@ describe('abacus-seqid', () => {
         'clearInterval'
       ]
     });
+    seqid = require('..');
   });
+
   afterEach(() => {
-    // Restore original timers
     clock.restore();
+    delete require.cache[require.resolve('..')];
   });
 
-  it('returns unique sequential ids', () => {
-    // Get an id, expect the id counter to be 0
-    const id1 = seqid();
-    expect(id1).to.match(/-0-0-0-0/);
-
-    // Get another for the same time, expect the id counter to increase,
-    // making the id greater than the previously obtained one
-    const id2 = seqid();
-    expect(id2).to.match(/-0-0-0-1/);
-    expect(id2 > id1).to.be.true;
-
-    // Get an id for a later time, expect the id counter to be 0, and the
-    // id to be greater than the previous one
-    clock.tick(100);
-    const id3 = seqid();
-    expect(id3).to.match(/-0-0-0-0/);
-    expect(id3 > id2).to.be.true;
-
-    // Get an id for an earlier time (simulating clock skew), expect the
-    // id time to still be the later time and the id counter to increase
-    // to make the id greater than the previous one
-    clock.tick(-200);
-    const id4 = seqid();
-    expect(id4).to.match(/-0-0-0-1/);
-    expect(id4 > id3).to.be.true;
+  // A function that returns sequential time-based ids.
+  describe('seqid', () => {
+    context('without timestamp', () => {
+      it('returns unique sequential ids', () => {
+        // Get an id, expect the id counter to be 0
+        const id1 = seqid();
+        expect(id1).to.match(/-0-0-0-0/);
+
+        // Get another for the same time, expect the id counter to increase,
+        // making the id greater than the previously obtained one
+        const id2 = seqid();
+        expect(id2).to.match(/-0-0-0-1/);
+        expect(id2 > id1).to.be.true;
```

Comment:
minor: We may change this with:

```
expect(id2).to.be.above(id1);
```

to follow convention defined in the other test files

Reply:
`seqid` returns string. `to.be.above` requires a number or a date.